src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -309,11 +307,8 @@ protected function prepareForDenormalization($data)
      * Returns the method to use to construct an object. This method must be either
      * the object constructor or static.
      *
-     * @param array            $data
-     * @param string           $class
-     * @param array            $context
-     * @param \ReflectionClass $reflectionClass
-     * @param array|bool       $allowedAttributes
+     * @param string     $class
```

Comment:
this should probably added as a type hint directly now, should I do it in the scope of this cleaning ?

Reply:
I'll add the type declaration with #33154.

btw, not all of the parameters are actually in use, as far as I can tell.

Reply:
Ok, noted, thanks, resolving conversation